.esh/commands/update-locales.py

```
@@ -1605,12 +1605,22 @@ def collect_placeholder_keys_recursive(data, prefix=""):
  * 表示所有可能的语言环境数据类型。
  */
 export type LocaleData = {locale_data_type_str}
+/**
+ * i18n switch 叶子（singular / plural 等），由 geti18n 按 params[switch] 解析。
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)